visualizations/bar_chart/webviz_bar_chart/__init__.py

```diff
@@ -14,22 +14,33 @@ class BarChart(FilteredPlotly):
         ``'overlay'``. Defines how multiple bars per index-value are combined.
         See `plotly.js layout-barmode <https://plot.ly/javascript/reference/
         #layout-barmode>`_.
-    :param xaxis: Will create a label for the x-axis. Defaults to `None`.
-    :param yaxis: Will create a label for the y-axis. Defaults to `None`.
+    :param xrange: list of minimum and maximum value. Ex: [3, 15].
+    :param yrange: list of minimum and maximum value. Ex: [3, 15].
+    :param xtitle: Will create a label for the x-axis. Defaults to `None`.
+    :param ytitle: Will create a label for the y-axis. Defaults to `None`.
```

Comment:
The other option here is to use `xlabel` and `ylabel` (which users coming from matplotlib are more used to). But at the same time `xtitle` and `ytitle` is close to plotly API. No strong opinion here, as long as it has been considered :)